tests/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ set (EXECUTABLE_OUTPUT_PATH bin/)
 #if(COMMAND cmake_policy)
 #    cmake_policy(SET CMP0003 NEW)
 #endif(COMMAND cmake_policy)
-
+add_definitions(-march=native)
```

Comment:
I think you should avoid to put compiler specific options in the CMakeLists.txt file. For instance, the tests on Visual Studio compiler (MSVC) will not pass.

Reply:
Ok. Reason for this line was that by default GCC uses the lowest architecture (SSE) and I'm unsure whether to add unsigned functionality here...

Reply:
If someone wants to compile in `SSE2` then MIPP will simply throw an exception saying that the called function using `uint32_t` is not available for this instruction set.

Reply:
As expected, the CI fails with `-march=native` : https://gitlab.com/aff3ct/MIPP/-/jobs/380213921

Reply:
Yep... I've already changed my local configuration and planed to push it today.